src/device-registry/utils/create-location.js

```diff
@@ -121,9 +119,7 @@ const createLocation = {
           data: responseFromModifyLocation.data,
           status,
         };
```

Comment:
just housekeeping to avoid a potential runtime error caused by running if statements without elseif